examples/component-versioning/component-mutable.md

```diff
@@ -37,6 +39,13 @@ data:
     namespace: default
     ...
   spec:
+    parameters:
+    - fieldPaths:
+      - metadata.name
+      name: instance-name
+    - fieldPaths:
+      - spec.containers[0].image
+      name: image
```

Comment:
I don't think these parameters are needed. Instead, using component revisioning discourage users using parameters.

Reply:
I run the examples and get that result.  I think it comes from https://github.com/crossplane/oam-kubernetes-runtime/blob/55cd9dbe93cfeba4fe03dfd8c81c34a4f367d2bc/examples/containerized-workload/sample_component.yaml#L19

Reply:
I see, we should copy the sample_component yaml from containerized-workload and make sure the new one without the parameters section. The the image tag below won't changed.

Reply:
`containerized-workload/sample_component.yaml` has this `parameters` section,  so `copy` won't work.  should we delete that code  from `containerized-workload/sample_component.yaml`  ?https://github.com/crossplane/oam-kubernetes-runtime/blob/55cd9dbe93cfeba4fe03dfd8c81c34a4f367d2bc/examples/containerized-workload/sample_component.yaml#L1-L26

Reply:
We can copy that, and remove the parameters section in the new example.

Reply:
fixed.